docker-compose.yml

```diff
@@ -61,11 +34,10 @@ slave2:
     - MESOS_ISOLATOR=cgroups/cpu,cgroups/mem
     - MESOS_LOG_DIR=/var/log
     - MESOS_RESOURCES=ports(*):[31000-31100]
-    - MESOS_MODULES=file:///isolator/build/isolator/modules.json
+    - MESOS_MODULES=file:///calico/modules.json
     - MESOS_ISOLATION=com_mesosphere_mesos_NetworkIsolator
     - MESOS_HOOKS=com_mesosphere_mesos_NetworkHook
     - ETCD_AUTHORITY=etcd:4001
-    - MESOS_ATTRIBUTES=public:true
```

Comment:
I think this will break the demo, since we target the "public" slave to launch the visualizer.